lib/Controller/SettingsController.php

```diff
@@ -22,54 +22,54 @@
 namespace OCA\PasswordPolicy\Controller;
 
 use OCP\AppFramework\Controller;
+use OCP\AppFramework\Http\DataResponse;
 use OCP\IConfig;
 use OCP\IRequest;
 use OCP\Settings\ISettings;
 use OCP\Template;
 
 class SettingsController extends Controller implements ISettings {
 
-	/** @var IRequest  */
-	protected $request;
 	/** @var IConfig  */
 	protected $config;
-	protected $appValues;
+
+	public const DEFAULTS = [
```

Comment:
visibility on const is php 7.1+ - intended? if so we need to adopt info.xml
I suggest to stay compatible with php 5.6

we need to add php-lint on php 5.6